commondata_projections_L0/FCCee_zh_240GeV.yaml

```diff
@@ -4,7 +4,7 @@ units: pb
 luminosity: 10000
 num_data: 1
 num_sys: 1
-data_central: 0.24007
+data_central: 0.1944
```

Comment:
In principle, if the central value changes, the stat uncertainties should change as well. Same for the other file.

Reply:
You're right, it'll be corrected.